openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -561,8 +561,7 @@ private Optional<MediaType> chooseResponseMediaType(ServerRequest req) {
     private static class HelidonAnnotationScannerExtension implements AnnotationScannerExtension {
 
         @Override
-        public Object parseExtension(String key, String value) {
-
+        public Object parseValue(String value) {
```

Comment:
I do not see why we need to create `parseValue` and use it from `parseExtension`. IIRC `parseValue` is a new method added to the SmallRye library in a later release (we use the later release in Helidon 4.x but not in 3.x). I notice that the new `parseValue` method here does not have `@Override` as it does in 4.x (which makes sense given that the method does not exist yet in SmallRye's library, at least from the Helidon 3.x perspective).

That said, the change seems to do no harm so I'm OK with it. If I have missed something about why it was added please let me know.

Reply:
When enabling test on `openapi-basic` example. It throws a class def not found exception about `jackson`.
In 3.x, the smallrye library version used is `2.1.16` and the `AnnotationScannerExtension` uses the `parseValue` with default implementation and uses `jackson`.
The new `parseValue` from my PR does have the `@Override` annotation ... line 563.
You can easily reproduce the issue by running `openapi-basic` example and enabling the test.